config/fam_pe_config.yaml

```diff
@@ -19,6 +19,9 @@ runtime: PMIX
 # client_interface_type value of direct implies communication happens via function calls.
 client_interface_type: rpc
 
+# Device interface for communication for PE node
+#pe_if_device: ib0 
```

Comment:
I think you can just give if_device. Also add a comment that it is valid only for verbs provider